plugins/gcov/gcov.rake

```diff
@@ -204,6 +204,19 @@ namespace UTILS_SYM do
       @ceedling[:tool_executor].exec(command[:line], command[:options])
     end
 
+    fail_under_line = @ceedling[:configurator].project_config_hash[:gcov_fail_under_line] || 0
+    fail_under_branch = @ceedling[:configurator].project_config_hash[:gcov_fail_under_branch] || 0
+    if fail_under_line + fail_under_branch > 0
+      puts "Checking the minimum coverage: #{fail_under_line}% by line and #{fail_under_branch}% by branch"
+      command = @ceedling[:tool_executor].build_command_line(TOOLS_GCOV_POST_FAIL_UNDER_COVERAGE, [], filter, fail_under_line, fail_under_branch)
+
+      begin
+        @ceedling[:tool_executor].exec(command[:line], command[:options])
```

Comment:
As far as I understand, this line generates new command that looks like:
```
gcovr -p -b -r "." -e "^vendor.*|^build.*|^test.*|^lib.*" --fail-under-line 85 --fail-under-branch 90 --html --html-details -o  "build_test/artifacts/gcov/GcovCoverageResults.html"
```

which results in a new report being generated. 

I think we should try to avoid generating report once again because:
1) it's just waste of effort
2) since not all possible parameters are taken into consideration in `gcov_post_fail_under_coverage`, this 2nd report might differ from the one generated previously, so from user perspective it will look like some of the settings didn't have any effect

Do you think it's possible instead of creating new command to modify all existing `gcov_post_report*` ones and optionally add limit checks (if configured) to all of them, so no matter which other gcov settings user chooses in Ceedling, limit checks will be included in all of them?

Reply:
Done